File: fibo/fibo.go
package main

import (
	"time"
	"fmt"
)

func fibo(num uint64) uint64{
	if num == 0 || num == 1 {
		return 1
	} else {
		return fibo(num - 1) + fibo(num - 2)
	}
}

func main(){
	start_timestamp := float64(time.Now().Unix()) + float64(time.Now().Nanosecond())/1e9
	fmt.Println(fibo(39))
	end_timestamp := float64(time.Now().Unix()) + float64(time.Now().Nanosecond())/1e9
	fmt.Println(end_timestamp - start_timestamp)
}

File: fibo/fibo.py
import time

def fibo(n):
    if n in (0, 1):
        return 1
    else:
        return fibo(n-1) + fibo(n-2)

if __name__ == '__main__':
    start = time.time()
    print(fibo(39))
    end = time.time()
    print(end - start)